test/cli_run_volume_test.go

```diff
@@ -64,6 +64,24 @@ func (suite *PouchRunVolumeSuite) TestRunWithLocalVolume(c *check.C) {
 		DefaultVolumeMountPath+"/"+funcname+"/test").Assert(c, icmd.Success)
 }
 
+// TestRunWithTmpFSVolume tests running container with tmpfs volume.
+func (suite *PouchRunVolumeSuite) TestRunWithTmpFSVolume(c *check.C) {
```

Comment:
Maybe add more commit for this test? since it's not the normal test for cli, more like a regression test? At least add a pr ref to help people understand this pr?